### PR TITLE
UX: Improve login required page

### PR DESCRIPTION
**Pull request by ellaestigoy**
**What changed in this PR?**
* Header menu is hidden
* Site description will be displayed if there's content
* If there's a wide logo uploaded, it will replace the 👋🏼 
* Removed both "An account is required. Please create an account or log in to continue." and "An account is required. Please ask an existing member for an invite or log in to continue." because it's obvious enough from the signup/login buttons
* Updated translations

### Before
image

### After
image

### CODE REVIEW

**Review by jordanvidrine — APPROVED** [review]

File: app/assets/javascripts/discourse/app/templates/static.hbs [hunk 1]
```diff
@@ -6,30 +6,32 @@
         <PluginOutlet @name="above-static" @connectorTagName="div" />
       </span>
 
-      {{html-safe this.model.html}}
+      <div class="login-welcome">
```
Comment (CvX) [inline app/assets/javascripts/discourse/app/templates/static.hbs:9]: @ellaestigoy this template is used for faq/guidelines/tos/privacy/etc pages. did you want to wrap their contents in `.login-welcome`?

File: app/assets/javascripts/discourse/app/controllers/static.js [hunk 2]
```diff
@@ -20,6 +21,9 @@ export default Controller.extend({
     );
   },
 
+  @discourseComputed
+  wavingHandURL: () => wavingHandURL(),
+
```
Comment (CvX) [inline app/assets/javascripts/discourse/app/controllers/static.js:26]: @ellaestigoy Where is this used?

Comment (ellaestigoy) [reply app/assets/javascripts/discourse/app/controllers/static.js:26]: The 👋🏼 is used when they don't have a logo uploaded

File: app/assets/javascripts/discourse/app/templates/static.hbs (hunk 1 above)
Comment (ellaestigoy) [reply app/assets/javascripts/discourse/app/templates/static.hbs:9]: Nah - only for this, as the CSS styling (rounded corners, box-shadow etc) is applied to this class

**Pull request merged**